website/src/client/components/shared/ResizeDetector.tsx

```diff
@@ -1,45 +1,19 @@
 import { StyleSheet, css } from 'aphrodite';
 import * as React from 'react';
+import { useResizeDetector } from 'react-resize-detector';
```

Comment:
Depends on the ResizeObserver web API, which might not be supported on older browsers.
https://www.caniuse.com/resizeobserver

It's possible to use a polyfill as well.
https://www.npmjs.com/package/react-resize-detector

@byCedric What do you think?

Reply:
I think this is fine, +91% global support. The biggest non-supported browser would be IE, but I think that's fine since Microsoft dropped support for IE as well. If this becomes a problem, we can always add a "Hey, your browser is outdated" message.

![Screenshot 2021-07-29 at 12 26 43](https://user-images.githubusercontent.com/1203991/127476353-57aacede-c46d-4558-9848-b319fc769c66.png)